stats/opentelemetry/client_metrics.go

```diff
@@ -155,12 +178,23 @@ func (csh *clientStatsHandler) processRPCEvent(ctx context.Context, s stats.RPCS
 		atomic.AddInt64(&mi.sentCompressedBytes, int64(st.CompressedLength))
 	case *stats.InPayload:
 		atomic.AddInt64(&mi.recvCompressedBytes, int64(st.CompressedLength))
+	case *stats.InHeader:
+		csh.getLabelsFromPluginOption(mi, st.Header)
```

Comment:
Maybe this should be `set` since it doesn't seem to `get` (return) anything?

Reply:
Switched.